### PR TITLE
Upgrade "eslint-config-skyscanner" and remove Prettier

### PR DESCRIPTION
**Pull request by sogame**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## UNRELEASED
 
-_Nothing yet..._
+### Changed
+- Bump `eslint-config-skyscanner` to `5.0.0`. Prettier is now included in `eslint-config-skyscanner`, and this package is just a wrapper around it. It's recommended to switch to `eslint-config-skyscanner` as soon as possible, as `eslint-config-skyscanner-with-prettier` will be deprecated soon.
```
Comment (olliecurtis) [inline CHANGELOG.md]: Rather than say deprecated soon, should we just say it's deprecated now and soon to be archived?

**Comment by olliecurtis** [comment]
@sogame will you al be able to bump the version in the package.json for when we release 🙂

File: CHANGELOG.md (hunk 1 above)
Comment (sogame) [reply CHANGELOG.md]: Makes total sense, changed here and in readme.md

**Comment by sogame** [comment]
Version bumped in `package.json` and `package-lock.json`, sorry, I though this was done automatically as part of the release 😅

**Pull request merged**